src/client/runner/automation/playback/move.js

```diff
@@ -1,355 +1,362 @@
+            };
+        }
+
+        var clientPosition = positionUtils.getClientPosition(this.element);
+        var isDocumentBody = this.element.tagName && this.element.tagName.toLowerCase() === 'body';
+
+        return {
+            x: isDocumentBody ? clientPosition.x + this.offsetX : clientPosition.x + this.offsetX - scroll.left,
+            y: isDocumentBody ? clientPosition.y + this.offsetY : clientPosition.y + this.offsetY - scroll.top
+        };
+    }
+
+    _emulateEvents () {
+        var currentElement = this.dragMode ? this.dragElement : getElementUnderCursor();
+        var whichButton    = this.dragMode ? eventUtils.WHICH_PARAMETER.leftButton : eventUtils.WHICH_PARAMETER.noButton;
+        var button         = this.dragMode ? eventUtils.BUTTONS_PARAMETER.leftButton : eventUtils.BUTTONS_PARAMETER.noButton;
+
+        var eventOptions = {
+            clientX: this.x,
+            clientY: this.y,
+            button:  0,
+            which:   browserUtils.isWebKit ? whichButton : 1,
+            buttons: button,
+            ctrl:    this.modifiers.ctrl,
+            alt:     this.modifiers.alt,
+            shift:   this.modifiers.shift,
+            meta:    this.modifiers.meta
+        };
+
+        // NOTE: if lastHoveredElement was in an iframe that has been removed, IE
+        // raises an exception when we try to compare it with the current element
+        var lastHoveredElementInDocument = lastHoveredElement &&
+                                           domUtils.isElementInDocument(lastHoveredElement);
+
+        var lastHoveredElementInRemovedIframe = lastHoveredElement &&
+                                                domUtils.isElementInIframe(lastHoveredElement) &&
+                                                !domUtils.getIframeByElement(lastHoveredElement);
+
+        if (lastHoveredElementInRemovedIframe || !lastHoveredElementInDocument)
+            lastHoveredElement = null;
+
+        var currentElementChanged = currentElement !== lastHoveredElement;
+
+        if (currentElementChanged && lastHoveredElement)
+            eventSimulator.mouseout(lastHoveredElement, extend({ relatedTarget: currentElement }, eventOptions));
+
+        //NOTE: only in IE a 'mousemove' event is raised before a 'mouseover' one (B236966)
+        if (browserUtils.isIE && currentElement)
+            eventSimulator[this.moveEvent](currentElement, eventOptions);
+
+        if (currentElementChanged) {
+            if (currentElement)
+                eventSimulator.mouseover(currentElement, extend({ relatedTarget: lastHoveredElement }, eventOptions));
+
+            lastHoveredElement = currentElement;
+        }
 
+        if (!browserUtils.isIE && currentElement)
+            eventSimulator[this.moveEvent](currentElement, eventOptions);
+
+        //NOTE: we need add extra 'mousemove' if element was changed  because sometimes
+        // client script require several 'mousemove' events for element (T246904)
```

Comment:
```
        //NOTE: we need to add an extra 'mousemove' if the element was changed because sometimes
        // the client script requires several 'mousemove' events for an element (T246904)
```